### DISCUSSION TITLE
happy april fools!

### DISCUSSION THREAD

**Discussion by raynepaws** [cytrinc/bark-coding#77] [General]
i added a button to the header that only shows on afd.... check it out
![Screenshot from 2024-04-01 13-01-39](https://github.com/mariocraft987/bark.github.io/assets/129226914/ff50fff9-985f-40bb-8f85-e433f210e2db)
![Screenshot from 2024-04-01 13-01-56](https://github.com/mariocraft987/bark.github.io/assets/129226914/6e1883a0-74fe-4510-9c32-1fbcd63727b1)

**Comment by mariocraft987** [comment]
Hehe
